### PR TITLE
Add support for local image paths in Img tags

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: lib/src/builder.dart [hunk 1]
```diff
@@ -244,7 +249,12 @@ class MarkdownBuilder implements md.NodeVisitor {
       }
     }
 
-    return new Image.network(path, width: width, height: height);
+    if (path.startsWith('http')) {
```
Comment (Hixie) [inline lib/src/builder.dart]: This isn't a good way to tell if it's a local vs remote image, consider if it was "http.png". Can we parse it using the URL parser and check the scheme?

Comment (mit-mit) [reply lib/src/builder.dart]: Ah, good point. Yes, let me parse with `Uri.parse` and check the scheme.

**Comment by mit-mit** [comment]
cc @Hixie fixed that bug, PTAL

File: lib/src/builder.dart [hunk 2]
```diff
@@ -244,7 +250,15 @@ class MarkdownBuilder implements md.NodeVisitor {
       }
     }
 
-    return new Image.network(path, width: width, height: height);
+    Uri uri = Uri.parse(path);
+    if (uri.scheme == 'http' || uri.scheme == 'https') {
+      return new Image.network(uri.toString(), width: width, height: height);
```
Comment (Hixie) [inline lib/src/builder.dart]: not ftp? :-)

I think we should probably go with Image.network if `uri.hasScheme` is true, regardless of the scheme.

**Comment by Hixie** [comment]
LGTM

File: lib/src/builder.dart (hunk 2 above)
Comment (Hixie) [reply lib/src/builder.dart]: LGTM either way

Comment (mit-mit) [reply lib/src/builder.dart]: `uri.hasScheme` doesn't work as this returns true:

```
void main() {
  Uri uri = Uri.parse('file://file.zip');
  print (uri.hasScheme);
}
```

Comment (Hixie) [reply lib/src/builder.dart]: Maybe hard-code "file" as the scheme for which to use the file logic instead? Maybe we should document that this only works for http: and https: and file paths? I could see someone wanting it to work for asset names, for instance (that won't work with an Image.file right?).

Comment (mit-mit) [reply lib/src/builder.dart]: I don't think there is much tradition for using `file://` in markdown, so would rather not do that. But, yes, we should document! PR here: https://github.com/flutter/flutter_markdown/pull/21

**Pull request merged**